orttraining/orttraining/test/training_api/common/synthetic_data_loader.h

```diff
@@ -32,14 +33,6 @@ struct SyntheticInput {
     }
   }
 
-  virtual ~SyntheticInput() {}
-
-  template <typename T>
-  std::vector<T>& GetData() {
-    auto ptr = dynamic_cast<TypedSyntheticInput<T>*>(this);
-    return ptr->Data();
-  }
-
   size_t NumOfElements() {
```

Comment:
>) { [](http://example.com/codeflow?start=23&length=3)

This method can not be called on a const instance of the class.

Reply:
Thanks! Fixed it.